### ISSUE TITLE
Include Vulkan backend

### ISSUE THREAD

**Issue by yves-surrel** [vincefn/pyvkfft#4]
It would be nice to have also a wrapper for the "native" Vulkan backend.

Maybe this [Vulkan python wrapper](https://github.com/EthicalML/vulkan-kompute) could be of some help.

**Comment by vincefn** [comment]
Yes it would be interesting if the Vulkan backend could be accessible as well.

However I have no use for the Vulkan backend, so I can't work on this and it'll have to come from someone else.

**Comment by dcmvdbekerom** [comment]
I'd be interested in this as well, I don't have time at the moment but perhaps in some future..
